src/ir/context.rs

```diff
@@ -522,7 +525,7 @@ impl BindgenContext {
                 &index,
                 "",
                 &options.clang_args,
-                &options.input_unsaved_files,
+                &input_unsaved_files,
```

Comment:
The Github CI checks seem to think the `&` is not necessary here. Is it?
```suggestion
                input_unsaved_files,
```